submissions/HannaSyn/Popup/css/style.css

```diff
@@ -236,6 +223,11 @@ input:focus ~ label {
     content: "Less"; 
 } 
 
-.menu__input:checked ~ .menu__label::before {
+.menu__input:checked ~ .menu-popup::before {
     display: block;
 }
+
+.main{
```

Comment:
You can add this bg to the `<body>` and you receive some benefits:
- In your case you no need tag `<main>`
- you can remove `position: absolute;` and all lines for positioning from the block `.navigation` and the image still will be under the `.header` block.
less code - less time to full page load.

Reply:
done. But I have doubt to use this method in the real project.